### DISCUSSION TITLE
Duda de como extraer el tipo de pokemon

### DISCUSSION THREAD

**Discussion by Juan-2003** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#465] [Q&A] [answered]
Que onda chicos. Estoy tratando de extraer del API los datos necesarios para que me diga el tipo de pokemon que es (Agua, Fuego, etc). 
Pero no hayo la manera para hacerlo. Intente con un :

let tipo = data.types.type.name;
console.log(tipo)

Si me pudieran echar una mano. Se los agradecería.

**Comment by ErickMontesDK** [comment, marked as answer]
Lo que pasa es que no puedes acceder así ya que si ves en la pokeapi, en types la info esta contenida en un 0, luego types y name. 
Estos son vectores almacenados no puedes acceder a la información asi.
Puedes entrar a data.types, luego entras a la seccion 0 con types[0], o sea al elemento 0 en types.
Luego ya entras con type.name.
Ahi esta de ejemplo mi codigo
![image](https://user-images.githubusercontent.com/99063569/159136615-314b7ba6-eaa3-489f-bd56-9d3dbab67f22.png)
